src/interface_adapter/create_game/CreateGamePresenter.java

```diff
@@ -30,8 +32,31 @@ public void prepareFirstRoundView(CreateGameOutputData createGameOutputData) {
         roundState.setCurrentLives(createGameOutputData.getLives());
         roundState.setInitialLives(createGameOutputData.getLives());
 
+        if (createGameOutputData.getDifficulty().equalsIgnoreCase("easy")) {
+            roundState.setEasyRound(true);
+            roundState.setMediumRound(false);
+            roundState.setHardRound(false);
+            roundState.setMultipleChoice1(createGameOutputData.getMultipleChoiceAnswers().get(0));
+            roundState.setMultipleChoice2(createGameOutputData.getMultipleChoiceAnswers().get(1));
+        }
+        else if (createGameOutputData.getDifficulty().equalsIgnoreCase("Medium")) {
+            roundState.setEasyRound(false);
+            roundState.setMediumRound(true);
+            roundState.setHardRound(false);
+            roundState.setMultipleChoice1(createGameOutputData.getMultipleChoiceAnswers().get(0));
+            roundState.setMultipleChoice2(createGameOutputData.getMultipleChoiceAnswers().get(1));
+            roundState.setMultipleChoice3(createGameOutputData.getMultipleChoiceAnswers().get(2));
+            roundState.setMultipleChoice4(createGameOutputData.getMultipleChoiceAnswers().get(3));
```

Comment:
```suggestion
            roundState.setMultipleChoiceOptions(createGameOutputData.getMultipleChoiceAnswers());
```
Continuation of cleaning the round state